tests/python/contrib/test_ethosu/test_legalize.py

```diff
@@ -221,135 +221,135 @@ def get_shape_expr(in_expr, out_expr):
     return shape
 
 
+def compute_ofm_shape(ifm_shape, padding, kernel_shape, strides, dilation):
+    if padding.lower() == "valid":
+        h = math.ceil((ifm_shape[1] - (kernel_shape[0] - 1) * dilation[0]) / strides[0])
+        w = math.ceil((ifm_shape[2] - (kernel_shape[1] - 1) * dilation[1]) / strides[1])
+    if padding.lower() == "same":
+        h = math.ceil(ifm_shape[1] / strides[0])
+        w = math.ceil(ifm_shape[2] / strides[1])
+    ofm_shape = [ifm_shape[0], h, w, kernel_shape[3]]
+    return ofm_shape
```

Comment:
There is an identical function with the same name infra.py, you should use that one

Reply:
The specifics of not using compute_ofm_shape from infra.py escape me at the moment, but I believe it was to do with an assertion failing. Let me see if I can rectify it so I don't have to use the exact same function minus the assertion.

Reply:
Just wondering if there was a reason the function in `infra.py` didn't work?

Reply:
Ah apologies, slipped through. If I remember correct, it failed the assertion for kernel_length == 2, which was previously 4. Should now work. Updating to use the function in infra.py now. cc @lhutton1

Reply:
Nice :) thanks @dchauhan-arm